addon/marks/inline-rdfa.ts

```diff
@@ -1,37 +1,62 @@
-import type { Mark, MarkSpec } from 'prosemirror-model';
+import type { Mark } from 'prosemirror-model';
 import { getRdfaAttrs, rdfaAttrSpec } from '@lblod/ember-rdfa-editor';
-import { renderRdfaAware } from '../core/schema';
+import {
+  classicRdfaAttrSpec,
+  getClassicRdfaAttrs,
+  getRdfaContentElement,
+  renderRdfaAware,
+} from '../core/schema';
+import type SayMarkSpec from '../core/say-mark-spec';
 
-export const inline_rdfa: MarkSpec = {
-  attrs: {
-    ...rdfaAttrSpec,
-  },
-  group: 'rdfa',
-  excludes: '',
-  parseDOM: [
-    {
-      tag: 'span',
-      // default prio is 50, highest prio comes first, and this parserule should at least come after all other nodes
-      priority: 10,
-      getAttrs(node: string | HTMLElement) {
-        if (typeof node === 'string') {
+type Options = {
+  rdfaAware?: boolean;
+};
+
+export const inline_rdfa: (options?: Options) => SayMarkSpec = ({
```

Comment:
It's not clear to me how the rdfaAware version of this mark should be used, as I thought the rdfa editing UI required `inline_rdfa` to be a node. So we would need some way to either automatically switch between mark and node using config or prompt the user to move it if they change the config.

Reply:
We can also leave the mark as is (just a non-configurable node-spec), and only make the `inline_rdfa` node-spec configurable.

Reply:
True, though it makes the migration more manual... Maybe this is one of those times where we can't cleanly make everything automatic though.

Reply:
I deprecated the mark version with a message to use the node version instead.